pkg/storage/storage.go

```diff
@@ -125,17 +134,18 @@ func getPlatformStorage(listers *regopclient.Listers) (imageregistryv1.ImageRegi
 	}
 
 	switch infra.Status.PlatformStatus.Type {
-	case configapiv1.LibvirtPlatformType:
-		cfg.EmptyDir = &imageregistryv1.ImageRegistryConfigStorageEmptyDir{}
-	case configapiv1.BareMetalPlatformType:
-		// There is no specific known storage type available for a "baremetal"
-		// platform deployment at install time, so we default to EmptyDir to
-		// allow the installation to complete cleanly.  This must be
-		// re-configured to use a PVC post-install once a storage platform has
-		// been configured.  Note that the only supported use of the
-		// "baremetal" platform does include rook/ceph based storage, so
-		// EmptyDir will never be used in a production cluster.
-		cfg.EmptyDir = &imageregistryv1.ImageRegistryConfigStorageEmptyDir{}
+
+	// These are the platforms we don't configure any backend for, on these
+	// we should bootstrap the image registry as "Removed".
+	case configapiv1.BareMetalPlatformType,
+		configapiv1.OvirtPlatformType,
+		configapiv1.LibvirtPlatformType,
+		configapiv1.NonePlatformType,
+		configapiv1.VSpherePlatformType:
```

Comment:
`libvirt` may in fact support PVC with 10Gi of storage. We can revisit this in a separate PR.

Reply:
Are they going to provide a PVC with a default name and we are going to bootstrap using it?

Reply:
@ricardomaraschini they are going to reconfigure the bootstrapped registry config because by default we use EmptyDir for libvirt.